docs/architectural-decision-records/029-push-submissions-into-salesforce.md

```diff
@@ -0,0 +1,51 @@
+# Push Submission Data Into Salesforce
+
+In order for CMS users to be able to review submissions in Salesforce, all submission need to be in the Salesforce database. We will first get data into Salesforce by calling Salesforce APIS from app-api on every submission. Other data sync strategies may follow. 
+
+### Decision Drivers
+
```

Comment:
I suggest adding can be implemented by end of calendar year to the  list. We have serious constraints around time.